setup.py

```diff
@@ -257,6 +256,7 @@ def compile_KLU():
         "all": [
             "anytree>=2.4.3",
             "autograd>=1.2",
+            "pandas>=0.24",
```

Comment:
Should we also make a "pandas" group? It looks like users would be needing `pandas` more frequently, and installing all the dependencies to get `pandas` might not be nice.

Reply:
Yes, i feel it's a good idea. I'd create a group for `pandas` too & update the docs accordingly.